compiler/plc_project/src/project.rs

```diff
@@ -295,6 +295,19 @@ impl<S: SourceContainer> Project<S> {
     pub fn get_output_format(&self) -> FormatOption {
         self.format
     }
+
+    /// Returns the configuration used by this project to evaluate the diagnostics
+    /// If no configuration is available, the generic configuration is returned
+    pub fn get_diagnostic_configuration(&self) -> impl AsRef<str> {
+        todo!("Missing");
+        #[allow(unreachable_code)]
+        ""
+    }
```

Comment:
looks like this is not in use. you want to keep it anyway?

Reply:
I'm not sure how this slipped in, I might be missing a commit.